README.md

```diff
@@ -27,22 +27,35 @@ On Ubuntu 18.04, you can install those dependencies like this:
 apt-get install maven openjdk-8-jdk python python3 docker.io docker-compose virtualenv make openvswitch-switch linux-generic-hwe-18.04
 ```
 
-Note that your build user needs to be a member of the docker group for the build to work. Do that by adding the user to /etc/groups and logging out and back in again.
+#### Gradle
+You can either install Gradle, or use Gradle wrapper:
+ - Option 1 - Install Gradle (ensure that you have gradle 6.0 or later) - https://gradle.org/install/
 
+ - Option 2 - Use Gradle wrapper. The Kilda repository contains an instance of Gradle Wrapper 
+ which can be used straight from here without further installation.
+
+#### Docker
+Note that your build user needs to be a member of the docker group for the build to work. 
+Do that by adding the user to /etc/groups and logging out and back in again.
+
+#### Maven
 You also need to increase the maven RAM limit at least up to 1G.
 
 ```export MAVEN_OPTS="-Xmx1g -XX:MaxPermSize=128m"```
```

Comment:
is it still valid?

Reply:
Good question. We still have modules built by Maven: Project Floodlight, OpenFlowJ, GUI.